src/store/types.ts

```diff
@@ -208,4 +208,126 @@ export interface RootState {
   experiments: Partial<Record<ExperimentType, boolean>>;
   watsNewModalVersion: string;
   enabledChains: WalletIdNetworkMap<ChainId[]>;
+
+  nftAssets: NFTAssets,
+  nftAssetsNumber: number,
+  starredNFTs: NFTAsset[]
+}
+
+export type NFTAsset = {
```

Comment:
Why is this type defined in wallet-core? Doesn't it come from CAL/chainify? It should be imported from there

Reply:
?